scripts/apps/authoring/workqueue/workqueue.ts

```diff
@@ -283,6 +283,13 @@ function WorkqueueCtrl(
             return $rootScope.link('authoring', item);
         }
     };
+
+    $scope.$on('$locationChangeSuccess', () => {
+        $scope.isMultiedit = $route.current && $route.current._id === 'multiedit';
+        if ($scope.isMultiedit) {
+            $scope.active = null;
```

Comment:
What is `active` referring to? Can you rename it?

Reply:
it's the item which is opened in authoring. any suggestions?

Reply:
I call it `articleInEditMode` in the API

https://github.com/superdesk/superdesk-client-core/blob/develop/scripts/core/superdesk-api.d.ts#L894